### ISSUE TITLE
Configure aliases as subdomains - nginx/docker

### ISSUE THREAD

**Issue by rabble** [planetary-social/rooms-frontend#24]
This needs someone who knows docker and nginx

Requires:
- [ ] update the nginx config (which is generated by docker?) to allow aliases as subdomains
- [ ] this process should be automated in the ansible playbook for rooms, for any domain

**Comment by mplorentz** [comment]
The short answer for how to do it comes in two pieces:
1) Some nginx config lines that route all subdomains to the same host as the root domain. A good example of a complete config can be found [here](https://github.com/ssbc/go-ssb-room/blob/master/docs/files/example-nginx.conf).
2) Nginx must serve a wildcard SSL certificate. The example config I linked above does this too. Automatically requesting and renewing the certificate is tricker and something I haven't done before. We have an existing ticket for this: https://github.com/planetary-social/infrastructure/issues/28

**Comment by cooldracula** [comment]
This is addressed in [planetary-social/ansible-scripts #12](https://github.com/planetary-social/ansible-scripts/pull/12), though it isn't working fully yet. The config is set, but the flow is not.  I suspect it has to do with some routing confusion on my part between what needs to go to the go-ssb-room and what needs to go to the frontend.

**Comment by cooldracula** [comment]
I do need a bit of help with this! I have a routing configuration set up, but I am not sure how to actually make the aliases to test them.  At the moment, when a route is undefined or using the 8008 then nginx pushes it to the go-ssb-room figure out. I am not sure how a client is communicating with planetary.pro to be able to set up aliases or communicate the aliases.  With a better understanding of this, I think it will be easier for me to figure out the routing for it.

**Comment by cooldracula** [comment]
I looked at this with @mplorentz today, and think this may require more discussion, as it may not be a solved problem at the code level.

We were both uncertain how aliases were meant to be handled by the vue router in the front end.

For example: at the moment if you go to https://zach.planetary.pro it will show you my alias information.  It's the same page as going to https://planetary.pro/alias/zach. Both are served by go-ssb-room.

If you go to https://planetary.pro/profile/zach, you get an error message that we could not load the profile.  I was not sure if this is because it's expecting a feed id instead of an alias, or if it was due to my profile having the opt-out of web publishing toggle set. 

What is the expected behaviour for `/profile/$alias` routes, in terms of vue's routing logic?  Does the vue frontend map the alias to the proper feed id through a graphql query or some other method? Or is it needing to communicate to go-ssb-room in some way?

This may be working already, but we'd want to test it with a public user.  If not,  would y'all be able to elaborate on what's expected on the vue side?

Thank you!

cc: @chereseeriepa @mixmix
